vcf_view.wdl

```diff
@@ -12,6 +12,7 @@ workflow xVCFViewWorkflow {
         Int cpu = 8
         Int memory = 64
         Int preemptible = 0
+        Int addl_disk = 1
```

Comment:
Nitpick: Consider un-abbreviating `addl`. I found it briefly confusing while I scoured my mind for possibilities.